NEWS

```diff
@@ -48,6 +48,12 @@
 		- Removing redundant and fixing implicit imports [Thoralf Klein]
 		- Hide many methods from SWIG, reducing compile memory by 500MiB [Heiko Strathmann, Fernando Iglesias, Thoralf Klein]
 
+2014-12-16 Fernando Iglesias <fernando.iglesias@shogun-toolbox.org>
```

Comment:
Could you also add the "new" news section? This is the one of the old release...